subprojects/docs/src/docs/userguide/getting-started.adoc

```diff
@@ -1,6 +0,0 @@
-include::installation.adoc[]
-
-[[sec:installation_getting_help]]
-=== Getting help
```

Comment:
OK. Can we keep them separate? I'm assuming that the role of the two documents is different. I could reinstate _getting-started.adoc_, although it would be confusing having both under _src/docs/userguide_. I don't really understand why a file that goes into the distribution is under _src/docs/userguide_ unless it's the user manual itself.